cluster.go

```diff
@@ -229,7 +229,7 @@ func (clstr *Cluster) tend() error {
 func (clstr *Cluster) waitTillStabilized() {
 	count := -1
 
-	doneCh := make(chan bool)
+	doneCh := make(chan bool, 1)
```

Comment:
These two are by definition equivalent. What is the issue that you're trying to solve by this change?